### ISSUE TITLE
Torch-serve not working, Worker Died was shown when I install torch 1.11 (stable)

### ISSUE THREAD

**Issue by m-ibrahim-khalil** [townim-faisal/serve-model-with-torchserve#1]
When I installed torch==1.8.2+cpu it worked fine.
I think, the requirements to run these models locally  are:

captum==0.5.0
certifi==2021.10.8
charset-normalizer==2.0.12
click==8.0.4
cycler==0.11.0
enum-compat==0.0.3
filelock==3.6.0
fonttools==4.30.0
future==0.18.2
huggingface-hub==0.4.0
idna==3.3
joblib==1.1.0
kiwisolver==1.4.0
matplotlib==3.5.1
numpy==1.22.3
packaging==21.3
pandas==1.4.1
Pillow==9.0.1
psutil==5.9.0
pyparsing==3.0.7
python-dateutil==2.8.2
pytz==2021.3
PyYAML==6.0
regex==2022.3.15
requests==2.27.1
sacremoses==0.0.49
scikit-learn==1.0.2
scipy==1.8.0
six==1.16.0
sklearn==0.0
threadpoolctl==3.1.0
tokenizers==0.11.6
torch==1.8.2+cpu
torch-model-archiver==0.5.3
torch-workflow-archiver==0.2.3
torchaudio==0.8.2
torchserve==0.5.3
torchvision==0.9.2+cpu
tqdm==4.63.0
transformers==4.17.0
typing-extensions==4.1.1
urllib3==1.26.9
